static/sass/main.scss

```diff
@@ -427,6 +427,11 @@ footer {
 
 }
 
+//Horizontal Properties Row should be visible only in narrow view ports
+@media only screen and (min-width: 700px){
+  #properties-row{display: none}
```

Comment:
This could be useful on other pages too.  Let's generalize the name of this element ID.  Let's try something like #horizontal-sub-nav.

Reply:
Done